views/me.hbs

```diff
@@ -43,42 +43,44 @@
 
             <a href="/dashboard/me/settings"><button class="settings">Settings</button></a>
             <a href="/logout"><button class="logout">Log Out</button></a>
-        </div>
+        </div> --}}
 
         <!-- APPLICATION STATUS -->
-        <div class='background theme-yellow container text-center'>
-            <h1>application status</h1>
+        <div class='background container text-center'>
+            <h1 class="app-title">application status</h1>
+            <hr class="line">
 
             <div class="status-container">
                 {{#if user.hasApplication}}
-                    <div class="status" style="background-color: {{statusColor}};">{{user.applicationStatus}}</div>
+                    <h2 class="description">Your application is under review!</h2>
+                    <div class="status pending">
+                        <h2 class="stats">{{user.applicationStatus}}</h3>
```

Comment:
Just want to point out that you have mismatched tags here (opens as `h2`, closes as `h3`). This happens in every copy of this HTML, so be sure to fix all the ones that remain after you clean up your `if/else` stuff